src/component/SymbolLogo.tsx

```
@@ -0,0 +1,30 @@
+import type { IconProps } from "./Icon/types";
+
+const SymbolLogo = ({ size = 42, className }: IconProps) => {
+  return (
+    <svg
+      width={size}
+      height="20"
+      viewBox="0 0 42 20"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      className={className}
+    >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)